lib/valueSetListener.js

```diff
@@ -257,6 +257,11 @@ class ValueSetImporter extends SHRValueSetParserListener {
       return;
     }
 
+    if (this._specs.valueSets._nsMap.has(this._currentNs)) {
+      if (this._specs.valueSets._nsMap.get(this._currentNs).has(this._currentDef.identifier._name)) {
```

Comment:
Same as noted above for the data elements -- try to find a way that avoids using `ValueSetSpecifications`'s private `_nsMap` property.  There should be a very similar solution available.